spec/services/metadata_cloud_service_spec.rb

```diff
@@ -92,7 +92,7 @@
       time_stamp_before = File.mtime(path_to_example_file.to_s)
       described_class.refresh_fixture_data(short_oid_path, metadata_source)
       time_stamp_after = File.mtime(path_to_example_file.to_s)
-      expect(time_stamp_before).to be < time_stamp_after
+      expect(time_stamp_before).to be <= time_stamp_after
```

Comment:
time_stamp before and after are the same after our changes. Do we need to look into this further?